src/i_video.c

```diff
@@ -1371,7 +1395,15 @@ void I_ResetScreen(void)
 
 void I_ShutdownGraphics(void)
 {
-    SDL_GetWindowPosition(screen, &window_position_x, &window_position_y);
+    if (fullscreen && exclusive_fullscreen)
+    {
+        window_position_x = 0;
+        window_position_y = 0;
```

Comment:
So we "forget" the position of the window if the user turns on exclusive fullscreen mode and exits the game? I'm fine with that, but maybe we can just ignore the `window_position_*` variables in exclusive fullscreen mode?

Reply:
Fixed by https://github.com/fabiangreffrath/woof/pull/1089/commits/e2b2ddf3e9358b310114c694f3f222fbec44eeed